src/modules/dashboard/components/Extensions/ExtensionSetup.tsx

```diff
@@ -108,6 +112,13 @@ const ExtensionSetup = ({
     [colonyAddress, extensionId, initializationParams],
   );
 
+  const {
+    isWhitelistExtensionEnabled,
+    whitelistAddress,
+  } = useEnabledExtensions({
+    colonyAddress,
+  });
+
   const initialValues = useMemo(() => {
```

Comment:
There are a few missing dependencies in the dependency array according to the linter